### DISCUSSION TITLE
After 70 Days, GitHub Support Still Hasn't Responded

### DISCUSSION THREAD

**Discussion by ghost** [community/community#141101] [Other Feature Feedback, Questions, & Ideas]
Hello,

My account has been flagged, but I haven't done anything against the Terms of Service. I've **checked and rechecked** everything, and while my account is visible to me, it appears invisible to everyone else. All my content is still there, but for others, it's as if my account doesn't exist.

My account is https://github.com/rdWei, and I'm currently posting with this secondary account because my main account doesn't even show the discussions I open.

Does anyone know how much longer I might have to wait for this issue to be resolved? Or should I consider giving up on this account altogether?

**P.S.** _I use this account for work, so it's of utmost importance to me. I have many projects, some of which I care about deeply. **Thank you for taking the time to read this.**_

**Comment by muchenspace** [comment]
Me too, I still haven't found any solution

**Comment by ghostinhershell** [comment]
Hi @shutenM,

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! In the future, please only open 1 ticket per incident.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
